### ISSUE TITLE
Cookie not isolated in Collection Runner

### ISSUE THREAD

**Issue by gregoirepaqueron** [a85/POSTMan-Chrome-Extension#4312]
## App Details:

```
Postman for Linux
Version 6.0.9
linux 4.4.0-116-generic / x64
```

Hello,I don't think this is intentional, but when running a collection with the Collection Runner, i expect it to be totally isolated from my previous request in the app (except if i choose Environment of course). It looks like it uses Cookie from my previous request that i did not execute in the collection. 
Explain by exemple :

Running a normal call in app :
```
GET mydomain.fr/somepage
```
 return a Response with a `Header "Set-Cookie -> SESSIONID=54sq6d54qdq"`.

When i run my collection which is based on the same domain : 
```
POST mydomain.fr/login
```
Its send the Cookie stored in my previous request, and its actually causing me a bug to already have a cookie for the first request. I would be glad to have an answer if i missed something, or if it is a real bug :)

Thanks !

**Comment by pwistrand** [comment]
I'm also experiencing this issue with cookies being picked up from response to a request in the collection runner then being added to the request's cookie collection. I then have to manually delete the cookie before any further runs will be successfully. I hope this isn't by design as IMO violates test isolation. I'm using the native client on osx.

**Comment by gregoirepaqueron** [comment]
any up on this one ?

**Comment by pwistrand** [comment]
I wish

**Comment by vikiCoder** [comment]
Hello,

We have added the following options in collection runner to provide more control over cookies while running a collection:
- Run collection without using stored cookies
- Save cookies after collection run

These options are available in [Postman App version 7.1.0 or later](https://www.getpostman.com/downloads/). Below is a screenshot showing off these options in runner window.

Screenshot:

<img width="1440" alt="Screenshot 2019-06-12 at 3 46 18 PM" src="https://user-images.githubusercontent.com/16588032/59343432-7fdee380-8d29-11e9-97e6-a48f865f8be3.png">
